modules/hardware/x86_64-generic/kernel/host/configs/networking.config

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: CC-BY-SA-4.0
+CONFIG_INET=y
+CONFIG_PACKET=y
+CONFIG_BRIDGE=y
+CONFIG_NETDEVICES=y
+CONFIG_USB_USBNET=y
```

Comment:
@vunnyso I think `CONFIG_USB_USBNET` should go to common `debug.config` fragment. What do you think?

Reply:
We can move `CONFIG_USB_USBNET` to `debug.config` only concern is  `CONFIG_USB_USBNET`  depends on `Depends on: NETDEVICES [=y] && USB_NET_DRIVERS [=y] and further depends on NET [=y]`, due to strong dependency on `networking.config` I have currently kept it under same file.

Reply:
That way the dependency is ok. If networking.config is left out the build and platform functionality will break anyway. But if debug.config is left out, it will harden the configuration.